providers/curation/github.js

```diff
@@ -109,13 +109,38 @@ class GitHubCurationService {
     // Github requires name/email to set committer
     if ((info.name || info.login) && info.email)
       fileBody.committer = { name: info.name || info.login, email: info.email }
+    else {
+      logger.info('Github requires name/email to set committer, info.email = ', info.email)
+    }
     if (get(currentContent, '_origin.sha')) {
       fileBody.sha = currentContent._origin.sha
+      // Extract the component values
+      const { coordinates, revisions } = currentContent
+      const { name, provider, type } = coordinates
+      let keys = Object.keys(revisions)
+      keys.forEach(revision => {
+        let component = name + '/' + revision
+        if(!this._validateComponentExists(component)){
+          //handle error
+          error => logger.info(`Failed to validate component exists ${component.toString()}: ${error.toString()}`)
+          return error;
```

Comment:
I'm not sure that a forEach statement is the right way to do this, since a curation could be done only for a specific revision. 
So i would suggest you to use something like https://lodash.com/docs/4.17.11#first , e.g. `_.first(revisions)`

Reply:
Yeah I had thought so too but const currentContent = await this._getCurations(coordinates) contains 2 revisions. I guess this is overkill so instead I should get the newContent and append it to the other coords.